.husky/pre-commit

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint
-yarn pretty
+# yarn lint
```

Comment:
**question:** Why commands are disabled?

Reply:
oh, forgot about it - I had to do it when you asked me to push it a few days ago - couldn't do it with 'yarn lint' active